### PR TITLE
docs(popover): new component usage and style tab content

### PR DESCRIPTION
**Pull request by laurenmrice**
**New**

Usage tab:
- Usage documentation
- Usage images

Style tab:
- Style documentation
- Style images

---

**Testing**
- Go to `Components` --> `Popover`
  - Review Usage tab
  - Review Style tab

### CODE REVIEW

File: src/pages/components/popover/style.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: Popover
+description:
+A popover is a layer that pops up over all other elements on a page.
```
Comment (theiliad) [inline src/pages/components/popover/style.mdx:4]:
```suggestion
description: A popover is a layer that pops up over all other elements on a page.
```

**Review by theiliad — CHANGES_REQUESTED** [review]

**Review by jeanservaas — APPROVED** [review]
Looks great!

**Comment by laurenmrice** [comment]
Ready for one more review from dev when you get a chance @joshblack or @dakahn  👀

**Review by dakahn — APPROVED** [review]

**Review by theiliad — APPROVED** [review]
LGTM

**Pull request merged**